heudiconv/tests/test_utils.py

```diff
@@ -58,4 +66,27 @@ def test_json_dumps_pretty():
           'Mar  3 2017 10:46:13 by eja'
     # just the date which reveals the issue
     # tstr = 'Mar  3 2017 10:46:13 by eja'
-    assert pretty({'WipMemBlock': tstr}) == '{\n  "WipMemBlock": "%s"\n}' % tstr
\ No newline at end of file
+    assert pretty({'WipMemBlock': tstr}) == '{\n  "WipMemBlock": "%s"\n}' % tstr
+
+
+def test_load_json(tmp_path, capsys):
+    # test invalid json
+    icontent = u"I'm Jason Bourne"
+    ifname = "invalid.json"
+    invalid_json_file = tmp_path / ifname
+    invalid_json_file.write_text(icontent)
+
+    with pytest.raises(JSONDecodeError):
+        load_json(str(invalid_json_file))
+        captured = capsys.readouterr()
```

Comment:
I believe all the checks should be brought out of the context manager, or otherwise would not actually be executed since above `load_json` would raise an exception (to be caught by `pytest.raises`)